fdbmonitor/fdbmonitor.cpp

```diff
@@ -210,12 +210,13 @@ const char* get_value_multi(const CSimpleIni& ini, const char* key, ...) {
 				keyCopy.at(i) = '_';
 			}
 		}
-		while (!ret && (section = va_arg(ap, const char*))) {
+		va_list tempArg;
+		va_start(tempArg, key);
+		while (!ret && (section = va_arg(tempArg, const char*))) {
 			ret = ini.GetValue(section, keyCopy.c_str(), nullptr);
 		}
+		va_end(tempArg);
```

Comment:
I think this works, though, you could probably make this code a little simpler by checking for both options in a single loop rather than running two loops.

Reply:
Thanks! I pushed the the modification about simplifying the code